department-of-veterans-affairs/va.gov-cms
# VACMS-1523: Updating section view to limit to owner tid arg.

## Description
See #1523 

## Testing done
Visual

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/86406899-f37bc780-bc81-11ea-8095-46b0f3607a2e.png)

![image](https://user-images.githubusercontent.com/2404547/86406916-00002000-bc82-11ea-9781-03fb12cef50d.png)

![image](https://user-images.githubusercontent.com/2404547/86406928-055d6a80-bc82-11ea-9fe0-4a9ae3835fc6.png)

## QA steps
- As an admin, go to`section/veterans-health-administration/vamc-facilities/va-pittsburgh-health-care?title=&type=press_releases_listing&moderation_state=All` and visually verify that only one item appears, and the owner is Pittsburgh.
- Filter by multiple content types, confirming that when results appear, they are owned only by Pittsburgh
